runtime/interpreter/value.go

```diff
@@ -6121,29 +6193,63 @@ type UIntValue struct {
 	BigInt *big.Int
 }
 
-func NewUIntValueFromUint64(value uint64) UIntValue {
-	return NewUIntValueFromBigInt(new(big.Int).SetUint64(value))
+func NewUIntValueFromUint64(memoryGauge common.MemoryGauge, value uint64) UIntValue {
+	const uint64Size = int(unsafe.Sizeof(uint64(0)))
+	return NewUIntValueFromBigInt(
+		memoryGauge,
+		common.NewBigIntMemoryUsage(uint64Size),
```

Comment:
Similarly, `common.NewBigIntMemoryUsage(uint64Size)` could be calculated only once

Reply:
Also refactored in 05a3f20542f1f94c0a04a33b7107229f7829688c